### PR TITLE
Docs: add contributing guidelines

### PR DESCRIPTION
**Pull request by jamesramsay**
Specifically, documents commit message conventions which are enforced.

### CODE REVIEW

**Comment by omgaz** [comment]
Maybe add a small section about pull requests.

Open an issue first
PR's need to be squashed before merging

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Please follow our established coding conventions (with regards to formatting, etc), please see our [style guides](#style-guides) below.
+
+We strive for 100% test coverage, to run tests locally use `npm run test`.
+
+We love lint. It keeps things consistent and protects against human error.Linting will be run automatically after running `npm run test`. You can also run `npm run lint` on its own. See [commit conventions](#commit-conventions).
+
+Ensure that your effort is aligned with the project's roadmap by talking to the maintainers, especially if you are going to spend a lot of time on it.
+
+When in doubt, ask. We're here to help :dromedary_camel: :izakaya_lantern:
```
Comment (camjc) [inline CONTRIBUTING.md]: kill emojis

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+## Guidelines
+Make sure there's an issue open for any work you take on and intend to submit as a pull request - it helps core members review your concept and direction early and is a good way to discuss what you're planning to do.
+
+Please follow our established coding conventions (with regards to formatting, etc), please see our [style guides](#style-guides) below.
+
+We strive for 100% test coverage, to run tests locally use `npm run test`.
+
+We love lint. It keeps things consistent and protects against human error.Linting will be run automatically after running `npm run test`. You can also run `npm run lint` on its own. See [commit conventions](#commit-conventions).
```
Comment (camjc) [inline CONTRIBUTING.md]: better to run `npm run posttest` since that runs all linting not just eslint

Comment (camjc) [inline CONTRIBUTING.md]: missing space after `human error.`

**Comment by camjc** [comment]
LGTM. Merging.

**Pull request merged**